render-utils.js

```diff
@@ -0,0 +1,24 @@
+export function renderBooks(book) {
+    const bookContainer = document.createElement('a');
+    const bookInfo = document.createElement('p');
+
+    bookContainer.classList.add('book-style');
+
+    bookInfo.textContent = book.book_title;
```

Comment:
I think the `book.book_title `is not coming through, I just see empty rectangles on the home page so far. so close!